### ISSUE TITLE
Automated Post Fixing script

### ISSUE THREAD

**Issue by nikolov-tmw** [nikolov-tmw/multilingual-wp#3]
After a user decides to **delete** a language that has been active(not just deactivate the language), we need to go through all posts for that language and all of the original posts that they refer to and:
1. Update the original post - change the meta, so that the to-be-deleted post's ID is no longer there.
2. Delete the post with `wp_delete_post()`

Maybe all of this can happen on an AJAX driven page to which the user will be redirected after confirming the deletion. This way we will avoid time-outs. 

Basically the deletion of a language should clear all of the traces from that language.

Just deactivating the language should keep all of the information about the language(including the posts that were created in that language and therefore the custom meta's related to posts in this language).

**Comment by nikolov-tmw** [comment]
Closing for now. Whenever user needs to update the posts, a message will be displayed asking them to go to the Fix Posts page.
